Cargo.toml

```diff
@@ -26,7 +26,7 @@ async-fs = "1.6.0"
 async-trait = "0.1.60"
 axum = { version = "0.6.1", features = ["multipart", "headers"] }
 axum-server = { version = "0.4.4", features = ["tls-rustls"] }
-base64 = "0.20.0"
```

Comment:
We going down down down?

Reply:
yes, I upgraded that in workspace dependency branch and it broke few things. I will open another issue to upgrade this.